incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/view/OperationsView.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.binding.openapi.internal.view;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.agrona.collections.Object2ObjectHashMap;
+
+import io.aklivity.zilla.runtime.binding.openapi.internal.model.PathItem;
+
+public final class OperationsView
+{
+    private final Map<String, Map<String, OperationView>> operationsByPath;
+    private final boolean hasResponses;
+
+    private OperationsView(
```

Comment:
Please move all the private constructors / methods to the end of the class for better readability.